qiskit/circuit/library/standard_gates/r.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Rotation around an axis in x-y plane."""
+
+import math
+import numpy
+from qiskit.qasm import pi
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+
+class RGate(Gate):
```

Comment:
@ajavadia are you really happy with this name?

Reply:
let's call this 
`RXYPlane(theta, phi)`

just have to figure out compatibility with aqt backends. i'll open an issue.